Plumbing in GameServices

robertmclaws opened the pull request:
- Change to `nameof()` in the switch statement.
- Events have to be generated on the Interface too.
- Reorg the DownloadClient and Interface to follow naming conventions.
- Wire up PlayerService event registration and de-registration.
- Handle some events and start getting data into the bindable properties.

Commit:
Change to nameof() for the switch statement.

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into rwm-dev

Commit:
Starting to plumb in GameServices

- Events have to be generated on the Interface too.
- Reorg the DownloadClient and Interface to follow naming conventions.
- Wire up PlayerService event registration and de-registration.
- Handle some events and start getting data into the bindable
properties.

review-ninja commented:
<a href="https://app.review.ninja/PoGo-Devs/PoGo/pull/29" target="_blank"><img src="https://app.review.ninja/65663167/pull/29/badge" alt="ReviewNinja"/>

Commit:
Add client event processing to base class.

Commit:
Wiring in device-specifics, Part 1

Commit:
Renaming Windows app + refactoring

- Fix namespace collision preventing GameServices from leveraging
Windows APIs.
- HttpClient fix I pushed to 1.1 last week.
- PTC token fix documented in PtcAuthenticationProvider.cs

Commit:
PTC Auth token fix
